stdlib/tests/crypto/rpo.rs

```diff
@@ -274,3 +274,87 @@ fn test_squeeze_digest() {
 
     build_test!(even_words, &[]).expect_stack(&even_hash);
 }
+
+#[test]
+fn test_hash_memory() {
+    // hash fewer than 8 elements
+    let compute_inputs_hash_5 = "
+    use.std::crypto::hashes::rpo
+
+    begin
+        push.1.2.3.4.1000 mem_storew dropw
+        push.5.0.0.0.1001 mem_storew dropw
+
+        push.5.1000
+
+        exec.rpo::hash_memory
+    end
+    ";
+
+    #[rustfmt::skip]
+    let expected_hash: Vec<u64> = build_expected_hash(&[
+        1, 2, 3, 4, 5
+    ]).into_iter().map(|e| e.as_int()).collect();
+    build_test!(compute_inputs_hash_5, &[]).expect_stack(&expected_hash);
```

Comment:
For this and other tests in this function, could we put an extra value onto the stack to make sure it is preserved. For example, this program could be:
```
use.std::crypto::hashes::rpo
begin
    push.1.2.3.4.1000 mem_storew dropw
    push.5.0.0.0.1001 mem_storew dropw
    push.11

    push.5.1000
    exec.rpo::hash_memory
end
```
And then the expected stack would be `[HASH, 11]`.